### DISCUSSION TITLE
Does CMI include a way to prevent Creeper from exploding and Enderman from moving the cube

### DISCUSSION THREAD

**Discussion by 8MiYile** [Zrips/CMI#7633] [Q&A] [answered]
this

**Comment by CloudeLecaw** [comment, marked as answer]
These are two settings that your protection plugin should do.
PreventExplosion or PreventCreeperExplode
And NoEndermenGrief

well if you have no protection plugin hat can prevent this you also can handle these by a seperate plugin like this one
https://www.spigotmc.org/resources/no-enderman-grief2.71236/

at least this request is a bit out of cmis scope.
cmi is no plugin to prevent your world from damage this is the work of single plugins made for it or a protection plugin. cmi is no world manager or world protection manager
